docs/application/web/guides/multimedia/media-controller.md

```diff
@@ -43,13 +43,13 @@ The main features of the Media Controller API include:
 
   You can use the media controller server to [receive the search requests](#receiving-search-request) using the `setSearchRequestListener()` method, and return the `RequestReply` object to the client who sent the request.
 
-- Setting content type for the currently playing media
+- Setting content type
 
-  You can [set content type for the currently playing media](#setting-content-type-for-currently-playing-media) using the updatePlaybackContentType() server method.
+    You can [set the content type](#setting-content-type-for-currently-playing-media) for the currently playing media
```

Comment:
Line 48 and 52, please add a period at the end of the sentence.

```suggestion
You can [set the content type](#setting-content-type-for-currently-playing-media) for the currently playing media.
```

Reply:
Done